dev-java/commons-jelly/commons-jelly-1.0.1.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Skeleton command:
+# java-ebuilder --generate-ebuild --workdir . --pom pom.xml --download-uri https://github.com/apache/commons-jelly/archive/refs/tags/commons-jelly-1.0.1.tar.gz --slot 1 --keywords "~amd64 ~x86" --ebuild commons-jelly-1.0.1.ebuild
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="commons-jelly:commons-jelly:1.0.1"
+JAVA_TESTING_FRAMEWORKS="junit junit-4"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="A Java and XML based scripting and processing engine"
+HOMEPAGE="https://commons.apache.org/proper/commons-jelly/"
+SRC_URI="https://github.com/apache/${PN}/archive/refs/tags/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="1"
+KEYWORDS="~amd64 ~x86"
+
+# Common dependencies
+# POM: pom.xml
+# commons-beanutils:commons-beanutils:1.6 -> >=dev-java/commons-beanutils-1.9.2:1.7
+# commons-cli:commons-cli:1.0 -> >=dev-java/commons-cli-1.4:1
+# commons-collections:commons-collections:2.1 -> >=dev-java/commons-collections-4.1:4
+# commons-discovery:commons-discovery:20030211.213356 -> >=dev-java/commons-discovery-0.5:0
+# commons-jexl:commons-jexl:1.0 -> >=dev-java/commons-jexl-2.1.1:2
+# commons-lang:commons-lang:2.0 -> >=dev-java/commons-lang-2.6:2.1
+# commons-logging:commons-logging:1.0.3 -> >=dev-java/commons-logging-1.2:0
+# dom4j:dom4j:1.5.2 -> >=dev-java/dom4j-1.6.1:1
+# forehead:forehead:1.0-beta-5 -> >=dev-java/forehead-1.0_beta5:0
+# jaxen:jaxen:1.1-beta-4 -> >=dev-java/jaxen-1.2.0:1.2
+# jstl:jstl:1.0.6 -> >=dev-java/tomcat-jstl-spec-1.2.5:0
+# junit:junit:3.8.1 -> >=dev-java/junit-3.8.2:0
+# servletapi:servletapi:2.3 -> java-virtuals/servlet-api:3.0
+# xerces:xercesImpl:2.12.0 -> >=dev-java/xerces-2.12.0:2
+# xml-apis:xml-apis:1.0.b2 -> !!!groupId-not-found!!!
+
+CDEPEND="
+	dev-java/commons-beanutils:1.7
+	dev-java/commons-cli:1
+	dev-java/commons-collections:0
+	dev-java/commons-discovery:0
+	dev-java/commons-jexl:1.0
+	dev-java/commons-lang:2.1
+	dev-java/commons-logging:0
+	dev-java/dom4j:1
+	dev-java/forehead:0
+	dev-java/jaxen:1.2[dom4j]
+	dev-java/junit:0
+	dev-java/tomcat-jstl-spec:0
+	java-virtuals/servlet-api:3.0
+"
```

Comment:
did you try that none of the latest slots of each dep cannot be used? following are those that should be tested: `commons-lang` (we really want to avoid anything older than `:3.6` as it does not compile with java source set to `1.8`), `servlet-api` (here you can in fact depend on `dev-java/tomcat-servlet-api`, just avoid slot `5.0`, there was a change in package names and it is not backward compatible).

jfyi, `commons-collections` has different package names for slot `:0` and `:4` so that one would not work for sure. similar for `commons-jexl` and slots `:1.0` and `:2`.

Reply:
Keeping this information bookmarked